docs/topics/Data.md

```diff
@@ -1,5 +1,25 @@
 # Data
 
-If you are on the dev team, reach out to [Stage](https://osu.ppy.sh/users/8191845) for a dump. Otherwise, please wait for a public dump to be made available.
+The o!TR team maintains a publicly accessible archive of database replicas to support two key objectives:
 
-If you have a database dump, you can populate your database by following the [recovery instructions](Disaster-Recovery.md).
\ No newline at end of file
+1. Facilitating development and contributions to the official [o!TR repositories](https://github.com/osu-tournament-rating/).
+2. Ensuring interested parties can verify a tournament (which used o!TR to seed or filter registrants) did so in a legitimate manner.
+
+## Public Replicas
+
+Database replicas are updated weekly and stored in a Google Cloud storage bucket. **The public may browse available replicas [here](https://data.otr.stagec.xyz/).**
```

Comment:
do you want to specify a weekly time that they will be created and/or made available? doesn't have to be in this doc itself